lib/droplet_kit/error_handling_resourcable.rb

```diff
@@ -7,12 +7,6 @@ def self.included(base)
         case response.status
         when 200...299
           next
-        when 429
-          error = DropletKit::RateLimitReached.new("#{response.status}: #{response.body}")
-          error.limit = response.headers['RateLimit-Limit']
-          error.remaining = response.headers['RateLimit-Remaining']
-          error.reset_at = response.headers['RateLimit-Reset']
-          raise error
```

Comment:
I'm just realizing that I should probably keep this in, and still raise the error if the `Retry-After` header isn't present, which indicates that the hourly rate limit has been reached, and further requests should fail.

Reply:
Fixed.